src/otrs-popup/otrs-popup.provider.js

```diff
@@ -0,0 +1,21 @@
+angular.module("ovh-angular-otrs")
+    .provider("OtrsPopup", function () {
+        "use strict";
+
+        var self = this;
+        var baseUrlTickets = null;
+
+        self.setBaseUrlTickets = function (url) {
+            if (angular.isDefined(url) && angular.isString(url)) {
```

Comment:
Maybe we can check the pattern of the URL, something like this https://www.regextester.com/94502 or validator.js ;)